### ISSUE TITLE
usage-advenced doesn't work

### ISSUE THREAD

**Issue by furkan-celik** [plotly/dash-player#61]
Hi,

I am trying to use current time feature of this tool to control a custom timeline. However in the given example code, current time text doesn't get updated until I stop the video. Which is suboptimal and in [here](https://community.plotly.com/t/dash-player-custom-component-playing-and-controlling-your-videos-with-dash/12349) it is updated continuously. Can you help me to make it get updated as the video flows? 

Thank you

Dash-player==1.0.3
Dash==2.7.0
Flask==2.2.2
Plotly==5.11.0

I have tested this on both windows 10 and WSL2

**Comment by furkan-celik** [comment]
One update on this, my colleague said the code is working for him but it is not working in my machine. I have tried fresh environment, different machines, port forwarding from a linux machine but nothing works. On the browser console, I am seeing some reactplayer errors that says "TypeError: Cannot read properties of underfined (reading 'getDuration') at y value" on dash_player.v1_.min.js:1.

I have also tried different browsers such as Chrome, Edge, IE, Firefox but all suffers from the same issue. Is there any setting on my machine that you think can trigger this? I have also tried disabling cache on browsers but that doesn't work either. And more weirdly DASH callback monitor screen on debug tools show callback call status as SUCCESS and output information goes to the widget but I can't see any visual update until I pause the video.
